contrib/runners/orquesta_runner/orquesta_functions/st2kv.py

```diff
@@ -25,10 +25,12 @@
 LOG = logging.getLogger(__name__)
 
 
-def st2kv_(context, key, decrypt=False):
+def st2kv_(context, key, **kwargs):
```

Comment:
Can we explicitly declare keyword arguments here? Not a fan of ``**kwargs`` since it doesn't make it immediately obvious which arguments a function takes.

I assume you did that to being able to distinguish if a default value is specified or not. Another way to handle that would be to set a default value of this keyword argument to some special value.

Reply:
Yes. I did this to distinguish whether default value is actually provided and not to be confused if someone wants default to be `None`.  Use of kwargs is more elegant in this case IMO. But if this is what's blocking, then I can assign some bogus value to default.  I'll call it `KAMI_DONT_WANT_KWARGS`.  :P

Reply:
@Kami Looks like I won't be able to change to an explicit arg for `default`.  When I set the default to some constant value, there may be a bug in YAQL that tries to make the parameter strongly typed.  When I try to use the function in YAQL expression, YAQL returns error saying it cannot recognize the signature.  If I set `default=None`, there's o problem.  I also try this with Jinja and there's no problem setting `default="SOME_CONSTANTS"`.  So at this point, you'll have to live with using kwargs here.